projects/noloan2/app/src/main/java/hasadna/noloan2/MainActivity.java

```diff
@@ -102,15 +69,17 @@ public void onClick(View v) {
   }
   
   // Get a list of all SMS messages in the inbox.
-  private ArrayList<SMSmessage> getSmsList() {
-    ArrayList<SMSmessage> smsList = new ArrayList<>();
-    Cursor cursor = getContentResolver().query(Uri.parse("content://sms/inbox"), null, null, null, null);
+  private ArrayList<SmsMessage> getSmsList() {
+    ArrayList<SmsMessage> smsList = new ArrayList<>();
+    Cursor cursor = getContentResolver().query(Uri.parse("content://sms/"), null, null, null, null);
+    
+    Log.i("how meny smss", "" + cursor.getColumnCount());
```

Comment:
`s/how meny smss/SMS count: `

Reply:
Now i remember why i didn't push it earlier.
This function didn't give me all the SMSs on my phone for some reason and I wanted to check on it.

Reply:
Did you check it? It you want to leave it for later, let's add a TODO in the code.